scaleway/data_source_instance_ip_test.go

```diff
@@ -0,0 +1,44 @@
+package scaleway
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccScalewayDataSourceInstanceIP_Basic(t *testing.T) {
+	tt := NewTestTools(t)
+	defer tt.Cleanup()
+	ipregexp := regexp.MustCompile("(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])")
```

Comment:
Does it work for IPv6? Should we use native parsing functions? https://pkg.go.dev/net#ParseIP ?

Reply:
We even got some help from the validation package: https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk@v1.17.2/helper/validation#IsIPAddress